src/chef-server-ctl/bin/chef-server-ctl

```diff
@@ -288,7 +289,7 @@ log_level = if ENV["CSC_LOG_LEVEL"]
 
 ChefServerCtl::Log.level = log_level
 
-ctl = Omnibus::ChefServerCtl.new(cmd_name, true, "Chef Server")
+ctl = Omnibus::ChefServerCtl.new(cmd_name, true, "#{ChefUtils::Dist::Server::PRODUCT}")
```

Comment:
Yes. Note that this isn't an _exact_ replacement, the result becomes `Chef Infra Server` instead of `Chef Server`

If we prefer an exact replacement I could use:

```"#{ChefUtils::Dist::Infra::Short.capitalize} Server"```

Or we could add a new constant to `ChefUtils::Dist::Server`

Reply:
IMHO it is likely preferred to change it to `Chef Infra Server` so this is the correct constant.

Reply:
yeah it is only used as a display sing in https://github.com/chef/omnibus-ctl/blob/main/lib/omnibus-ctl.rb So it should be fine to use Chef Infra Server instead of Chef Server. Thank you for the fix!